src/main/java/org/shanerx/tradeshop/bitrade/BiTradeEventListener.java

```diff
@@ -60,12 +61,12 @@ public void onBlockInteract(PlayerInteractEvent e) {
             try {
                 chestState = findShopChest(s.getBlock()).getState();
             } catch (NullPointerException npe) {
-                buyer.sendMessage(colorize(getPrefix() + plugin.getMessages().getString("missing-shop")));
+                buyer.sendMessage(colorize(getPrefix() + Message.MISSING_SHOP.toString()));
```

Comment:
how about making Message.toString() prefix the string automatically? More efficient than using `getPrefix() + Message` every time (DRY again?)